SU-HSDO/stanford_mrc
# Mrcd8 176 visitor display

# READY FOR REVIEW

# Summary
- Node display settings for visitor content type.

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout branch
2. update database
3. create/edit visitor content
4. verify the correct fields and labels are displayed. [Mock Up](https://projects.invisionapp.com/share/DCEGPUZS4#/screens/265032969) (needs some styling from joe)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
